WhatBackend.EmailRenderService/WhatBackend.EmailRenderService/Properties/launchSettings.json

```diff
@@ -21,7 +21,7 @@
       "commandName": "Project",
       "launchBrowser": false,
       "launchUrl": "emailrenderservice",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "applicationUrl": "https://localhost:5001;http://localhost:5002",
```

Comment:
is it important?

Reply:
It was changed because both .Api project and EmailSenderService were taking 5000 port.
Was impossible to test it's work on local.